Services/Calendar/templates/default/tpl.day_view.html

```diff
@@ -60,4 +60,9 @@
 		<!-- END time_row -->
 	</tbody>
 </table>
-
+<script>
+    $('.calfullcontent').each(function(){
+        var fs = parseInt($(this).parent().height());
```

Comment:
I just tested this. Will not work if wrapped, since parent might be changed. You need to be more explicit here. This should be easily possible by some selector for the element you are actually looking for.